src/app/public/components/impressum/impressum.component.html

```diff
@@ -119,310 +121,217 @@ <h3 style="font-weight: bold; margin-top: 3em" class="u-text-h3-semibold u-text-
-				Löschung (Art. 17 DS-GVO), Einschränkung der Verarbeitung (Art. 18 DS-GVO) sowie auf Datenübertragung
-				(Art. 20 DS-GVO).
-			</span>
 			<span style="display: block; margin-bottom: 0.9em">
-				Ihr habt außerdem das Recht, Euch an eine Aufsichtsbehörde zu wenden (z.B. Die Landesbeauftragte für
-				Datenschutz und Informationsfreiheit Nordrhein-Westfalen, Kavalleriestr. 2-4, 40213 Düsseldorf).
-			</span>
-			<span style="display: block; margin-bottom: 0.9em; font-weight: bold">
-				Sofern personenbezogenen Daten auf Grundlage von Art. 6 Abs. 1 f) DS-GVO verarbeitet werden, besteht die
-				Möglichkeit, gegen die Datenverarbeitung zu widersprechen, soweit dafür Gründe vorliegen, die sich aus
-				Eurer besonderen Situation ergeben oder sich der Widerspruch gegen Direktwerbung richtet (Art. 21
-				DS-GVO). Bei einem Widerspruch gegen Direktwerbung werden Euch keine werblichen Nachrichten mehr
-				übermittelt.
-			</span>
+                Die Webseite gibt dir die Möglichkeit, sich über Open Educational Badges zu informieren und digitale Badges zu erhalten, zu sammeln, mit anderen zu teilen und zu vergeben. Wir verarbeiten deine Daten ausschließlich auf Basis der von dir getroffenen Einwilligung. Zweck der Verarbeitung ist es, unsere Webseite und Leistungen ordnungsgemäß darzustellen und anzubieten. Im Einzelnen können deine Daten wie folgt verarbeitet werden:
+            </span>
+		</p>
+        <h3 style="font-weight: bold; margin-top: 1.3em" class="u-text-h3-semibold u-text-dark2">
+            4.1 Rein informatorische Nutzung der Webseite
+        </h3>
+        <p style="font-size: 1.1rem; color: black; line-height: 1.55; margin: 0.5em 0 1.3em">
 			<span style="display: block; margin-bottom: 0.9em">
-				Ihr könnt eine gegenüber uns erteilte Einwilligung jederzeit mit Wirkung für die Zukunft widerrufen. Das
-				hat allerdings keinen Einfluss auf die bis dahin erfolgte Datenverarbeitung auf Basis dieser
-				Einwilligung (z.B. Versand eines Newsletters).
-			</span>
+                Besuchst du unseren Internetauftritt zu rein informatorischen Zwecken, ist es grundsätzlich nicht erforderlich, dass du uns Daten zur Verfügung stellst. In diesem Fall erheben und verwenden wir nur diejenigen Daten, die uns Ihr Internetbrowser automatisch übermittelt, wie zum Beispiel
+            </span>
 			<span style="display: block; margin-bottom: 0.9em">
-				Nutzt bitte die im Impressum angegebenen Kontaktdaten für die Geltendmachung Eurer Rechte.
-			</span>
-		</p>
+                <ul style="list-style: disc; list-style-position: inside; padding-left: 1.2em">
+                    <li>Datum und Uhrzeit des Abrufs einer unserer Internetseiten,</li>
+                    <li>Typ und Einstellungen des von Ihnen verwendeten Internetbrowsers,</li>
+                    <li>verwendetes Betriebssystem,</li>
+                    <li>die von Ihnen zuletzt besuchte Internetseite, Ihre IP-Adresse.</li>
```

Comment:
i believe "Ihre IP-Adresse" soudl be its own bullet point.

Reply:
Fixed it (and I also removed the sentences on image deletion)